lib/Runtime/Base/FunctionBody.cpp

```diff
@@ -6257,6 +6257,13 @@ namespace Js
         this->SetConstTable(nullptr);
         this->byteCodeBlock = nullptr;
 
+        // Also, remove the function body from the source info to prevent any further processing 
+        // of the function such as attempts to set breakpoints.
+        if (GetIsFuncRegistered())
```

Comment:
I don't think we do. In this particular case because it's OOM we will throw and come out of the EmitProgram call.